_docker-library/README.md

```diff
@@ -0,0 +1,10 @@
+# _docker-library
+
+This directory is a placeholder with samples for the following repos:
```

Comment:
I'm a bit confused, I thought we talked about *not* having samples here?

Reply:
These are not samples for how to use our repo. These are placeholder samples of what *we* need to submit to the other repos as part of the process of creating of publishing our official image.